presto-docs/src/main/sphinx/release/release-0.260.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.260
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug in SQL functions that would cause compiler error when session property ``inline_sql_function`` is set to ``false`` and SQL function references input in lambda expression.
+* Fix a bug in fragment result cache that caused query failures.
+* Add :ref:`uuid_type` type to represent UUIDs.
+* Add configuration property ``experimental.aggregation-spill-enabled`` and session property ``aggregation_spill_enabled`` to control aggregate spills explicitly.
+* Add configuration property ``experimental.order-by-spill-enabled`` and session property ``order_by_spill_enabled`` to control order by spills explicitly.
+* Add configuration property ``experimental.window-spill-enabled`` and session property ``window_spill_enabled`` to control window spills explicitly.
+* Add configuration property ``fragment-result-cache.max-single-pages-size`` to control the max fragement cache file size.
+* Add session property ``query_max_revocable_memory_per_node`` to override existing configuration property ``experimental.max-revocable-memory-per-node``.
+* Add configuration property ``optimizer.aggregation-if-to-filter-rewrite-enabled`` and session property ``aggregation_if_to_filter_rewrite_enabled`` to toggle an optimizer rule which improves the query performance of ``IF`` expressions inside aggregation functions.
+* Enable join spilling by default when spill is enabled.  This can be disabled by setting the configuration property ``experimental.join-spill-enabled`` or session property ``join_spill_enabled`` to ``false``.
+
+Hive Connector Changes
+______________________
+* Use Hive naming convention for bucket file names.
```

Comment:
This is unclear to me. Is there some documentation from Hive or something that can make this clear?